src/index.js

```diff
@@ -0,0 +1,32 @@
+import './style.css';
+
+const board = document.getElementById('board');
+
+const myScores = [{
+  name: 'Sean',
+  score: 100,
+},
+{
+  name: 'Dave',
+  score: 100,
+},
+{
+  name: 'Tshephang',
+  score: 100,
+}];
+
+function displayScores() {
```

Comment:
- [ ] Great job with the js, but one of the learning objectives for this project is to use ES6 syntax, which means that `function displayScores() {...}` should be an arrow function as in `const displayScores = () => {...}`

Reply:
Thank you I implemented the change

Reply:
This issue is well resolved! ✔️